### PR TITLE
Update: dokan-dev.Dokany version 1.5.0.3000

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20680)

### CODE REVIEW

File: manifests/d/dokan-dev/Dokany/1.5.0.3000/dokan-dev.Dokany.locale.en-US.yaml
Comment (OfficialEsco) [inline manifests/d/dokan-dev/Dokany/1.5.0.3000/dokan-dev.Dokany.locale.en-US.yaml:19]:
```suggestion
Publisher: Dokany Project
PublisherUrl: http://dokan-dev.github.io/
PublisherSupportUrl: https://github.com/dokan-dev/dokany/issues
#PrivacyUrl: 
Author: Dokany Project
PackageName: Dokany Library
PackageUrl: https://dokan-dev.github.io/
License: LGPL/MIT
LicenseUrl: https://github.com/dokan-dev/dokany/blob/master/license.mit.txt
Copyright: Copyright (C) 2007 Free Software Foundation, Inc. / Copyright (C) 2015 - 2020 Adrien J. and Maxime C.
CopyrightUrl: https://github.com/dokan-dev/dokany/blob/master/license.mit.txt
ShortDescription: User mode file system library for windows with FUSE Wrapper
Description: Dokany is a fork of Dokan 0.6.0 with bug fixes, clean change history and updated to build with latest tools.
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
image

**Comment by vedantmgoyal2009** [comment]
image

**Comment by OfficialEsco** [comment]
`Validation-Executable-Error` does not mean there is a issue with the Manifest which is why this is assigned to a technician.

**Comment by palenshus** [comment]
The issue here is that dokanctl.exe ends with exit code: 1, which the validator takes as a failure. Ideally the exe would return 0 when run with no arguments, otherwise we require a validation override.

File: manifests/d/dokan-dev/Dokany/1.5.0.3000/dokan-dev.Dokany.locale.en-US.yaml
Comment (ItzLevvie) [inline manifests/d/dokan-dev/Dokany/1.5.0.3000/dokan-dev.Dokany.locale.en-US.yaml:12]:
```suggestion
PackageName: Dokan Library
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by vedantmgoyal2009** [comment]
Please merge this PR.

**Comment by denelon** [comment]
@vedantmgoyal2009 I think this one is just waiting on a moderator approval. I removed the error label.

**Review by OfficialEsco — APPROVED** [review]
We are looking at `Azure-Pipeline-Passed` labels, not `Needs: Attention` or `Validation-Completed`
Edit: as you can see we have no power in this PR.

**Review by palenshus — APPROVED** [review]

**Comment by denelon** [comment]
It looks like @ItzLevvie still has changes requested on the PR.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**
